app/models/activity.rb

```diff
@@ -364,4 +364,8 @@ def is_newton_funded?
   def requires_country_delivery_partners?
     is_newton_funded? && programme?
   end
+
+  def status
```

Comment:
As this is only used in the IATI XML view, is `ActivityPresenter` a better place for it - maybe renamed to `iati_status`?

Reply:
I've renamed it, but it turns out that we don't use the presenter to show the activity in XML format, so it's still in the model.